examples/react-router-v4-example/src/components/Home.js

```diff
@@ -15,7 +16,7 @@ const Home = ({ actions: { logout }, user, authenticated }) => (
         >
           LOGOUT
         </button>
-      ))()
+      ))
```

Comment:
The Logout Button is not appearing anymore.

To fix it, I recommend to do a new component called `LogoutButton`

```javascript
import React from 'react';
import PropTypes from 'prop-types';
import { connect } from 'react-redux';
import { withRouter } from 'react-router-dom';
import { logout } from '../actions/sessionActions';

const LogoutButton = ({ history, logout }) => (
  <button
    onClick={() => logout(history)}
  >
    LOGOUT
  </button>
);

const { object, func } = PropTypes;

LogoutButton.propTypes = ({
  history: object.isRequired,
  logout: func.isRequired
});

const mapDispatch = dispatch => ({
  logout: history => dispatch(logout(history))
});

export default connect(null, mapDispatch)(withRouter(LogoutButton));
```

And then just use it in the `Home` component. If you find another way to fix it go ahead.